### DISCUSSION TITLE
TinyGLB: Simple 3D model hosting

### DISCUSSION THREAD

**Discussion by axbylabs** [google/model-viewer#4699] [Show and tell]
I used model-viewer recently to build [TinyGLB](https://tinyglb.com/), a free and simple way to host GLB models online.

There are a few options right now for customising the model page (switching materials, variants and mesh visibility), I hope to continue adding features if the tool proves to be useful to people.

Any feedback is welcome and thanks for the excellent model-viewer library!

![tinyglb](https://github.com/google/model-viewer/assets/162442194/3e6e2bc0-4010-46d9-a89d-56a457886ecc)

**Comment by elalish** [comment]
Thanks for sharing! If you want to put this on Twitter and @model-viewer, we'll happily repost it.
